src/LiveComponent/doc/index.rst

```diff
@@ -625,10 +625,33 @@ persisted entities, which dehydrate to an ``id``).
 Hydration, DTO's & the Serializer
```

Comment:
Let's rename this section to:

```
Using DTO's on a LiveProp
```

Then, below:

* Change `1) Hydrating with the Serializer` to `Hydrating with the Serializer` and change the `...` below it to be `~~~~` so it is a sibling of this section.
* Do the same for `2) Hydrating with Methods: hydrateWith & dehydrateWith`: remove the `2)` and change `...` to `~~~`